### DISCUSSION TITLE
A better tiddlywiki.com landing page

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#5496] [Ideas]
###  Part One
Opening https://tiddlywiki.com/ the HelloThere appears, it says

> Have you ever had the feeling that your head is not quite big enough to hold everything you need to remember?
> 
> Welcome to TiddlyWiki, a unique non-linear notebook for capturing, organising and sharing complex information.
> 
> Use it to keep your to-do list, to plan an essay or novel, or to organise your wedding. Record every thought that crosses your brain, or build a flexible and responsive website.

Considering TW 5.1.23 with a real unlimited use case, is the above description is suitable for TW 5.1.23 in year 2021? Definitely no!

### Part Two
Last year there was substantial discussion in Google Forum about Tiddlywiki landing page. This year we see some Tiddlywiki editions (or apps) have great landing pages while tiddlywiki.com still uses the old layout much more suitable for documentation and playground! I think it is the time to have a semantic, beautiful TW landing page!

Isn't it the time for change?

**Comment by clutterstack** [comment]
I agree about the landing page, with reservations. 

I think a simple and attractive landing page can be a boon. I also think many companies and projects go nuts with big graphics and lots of whitespace. I flinch when I land on a page like that and have to scroll past massive graphics to see each selling point in a massive font. It feels...full of hot air, lacking substance. Not trustworthy. So I guess I'm saying I worry about TiddlyWiki having a landing page that seems inauthentic and bandwagony. But there's nothing wrong with a bit of sprucing up.

[Vue.js](https://vuejs.org/) is a landing page that does a little sales pitch and prominently presents three main options: "Why Vue.js?", "Get Started", and "GitHub", along with a standard but unobtrusive top menu for people who know what they're looking for. (I don't like that "Why Vue.js" plays a video, but whatever.) What's behind "Get Started" is their normal documentation. Once you start using Vue, you probably never go to vuejs.org again. It's true that their pages are a bit cluttered with Patreon badges, but that has its purpose.

Maybe the whitespace and (somewhat) huge clip art could go on a "Why TiddlyWiki?" page.

My second hesitation is just a small reluctance to abandon the elegance of TiddlyWiki being so self-sufficient that it can be its own landing page. But when a pretty landing page can be hand-coded, perhaps I'm being stubborn and impractical.
